Documentation/MigrateToComposer/MigrationSteps.rst

```diff
@@ -43,40 +42,30 @@ However, this may require extensions you don't need or omit extensions you do
 need, so be sure to update the required extensions as described in the next
 sections.
 
-The :file:`composer.json` in the Base distribution includes a scripts section:
-
-.. code-block:: json
-   :caption: /composer.json
-
-   {
-      "scripts": {
-         "typo3-cms-scripts": [
-            "typo3cms install:generatepackagestates",
-            "typo3cms install:fixfolderstructure"
-         ],
-         "post-autoload-dump": [
-            "@typo3-cms-scripts"
-         ]
-      }
-   }
+You can also create the :file:`composer.json` file via a :bash:`composer init` command.
+Or use the `TYPO3 Composer helper <https://get.typo3.org/misc/composer/helper>`__.
+Also advanced project builders like `CPS-IT project-builder <https://github.com/CPS-IT/project-builder>`__
+help you to easily initialize this most vital file of a Composer project.
 
+.. hint::
 
-This requires helhum/typo3-console (so be sure to require that too) and is essential
-for generating the file :file:`typo3conf/PackageStates.php`.
-
+   Versions prior to TYPO3 v12 may reference a `scripts` section that would make use of
+   `helhum/typo3-console <https://packagist.org/packages/helhum/typo3-console>`__, and which
+   also would need to be required as a package in your newly created :file:`composer.json`.
 
-.. hint::
+   You can look at previous versions of the :file:`composer.json` in the mentioned repository,
+   if you want to migrate an older TYPO3 version to Composer.
```

Comment:
We should not recommend that. First: change to composer then upgrade or vice versa. Not all at once.